### DISCUSSION TITLE
Macbook USB mic no input and unable to find right settings

### DISCUSSION THREAD

**Discussion by FeeshUnofficial** [jamulussoftware/jamulus#1711] [Technical Support & Help] [answered]
My mother is trying to set up Jamulus on her Macbook Air. It's a slightly older model. I bought her a wired headset (which works) and a TONOR TC-777 USB microphone, a model which I have had a pleasant experience with up until this day on my own Windows PC, but doesn't seem to work on my mum's Macbook.

The input is getting through in the system preferences, but not in Jamulus. I tried following the guide for this problem at the bottom of the troubleshooting page, but we couldn't find `Microphone` on the left-hand side of `Security & Privacy` > `Privacy`

I am at a loss here, does anyone know how to fix this issue?

**Comment by ann0see** [comment, marked as answer]
Jamulus doesn’t detect microphone plugged in after you opened the App. Can you close Jamulus, plug in the mic re-open Jamulus and then check for the mic?
